assets/js/components/ProjectEditor.js

```diff
@@ -34,10 +34,22 @@ export function ProjectEditor (projectDescriptionCredits, programId, textFields,
     projectDescriptionCredits.data('trans-discard')
   )
 
+  this.confirmDeleteDialog = new ProgramEditorDialog(
+    projectDescriptionCredits.data('trans-confirm-delete'),
+    projectDescriptionCredits.data('trans-cancel'),
+    projectDescriptionCredits.data('trans-delete')
+  )
+
   this.translationUpdated = projectDescriptionCredits.data('trans-translation-updated')
+  this.translationDeleted = projectDescriptionCredits.data('trans-translation-deleted')
+  this.cannotDelete = projectDescriptionCredits.data('trans-cannot-delete')
```

Comment:
Why can't we hide the delete button for default language? plus, I thought the delete button is only visible when translation is defined. I don't see that code here.

Reply:
I was planning to do all the hiding and disabling the buttons in this ticket https://capstone-pm.atlassian.net/browse/SE-62 because I think the logic is going to change after the navigation is changed in this ticket https://capstone-pm.atlassian.net/browse/SE-61 that I was planning on doing next. Right now, this blocks them from deleting the default translation pretty easily and I will rework it once the navigation is reworked.

Reply:
remember to add a note on your pr about this